src/Cluster.jl

```diff
@@ -94,7 +94,7 @@ function init_centroids(X, K, mode)
     if !isa(K, Int) || K <= 0
         throw(ArgumentError("K must be a positive integer"))
     end
-    if mode != "kmeans" && mode != "kmeanspp"
+    if mode != "kmeans" && mode != "kmeanspp" && mode != "dc"
         throw(ArgumentError("mode must be either 'kmeans' or 'kmeanspp'"))
```

Comment:
Fix this line. Also maybe we should generally switch the mode back from string to symbol as Adrian said.